### PR TITLE
Feature: Main Layout

### PR DESCRIPTION
**Pull request by suezzo**
Added the base layout of the application. Refactored folder structure and created .editorconfig file.

### CODE REVIEW

File: .editorconfig [hunk 1]
```diff
@@ -0,0 +1,10 @@
+# editorconfig.org
+root = true
+
+[*]
+indent_style = space
+indent_size = 2
+end_of_line = lf
+charset = utf-8
+trim_trailing_whitespace = true
+insert_final_newline = true
```
Comment (stanimirtt) [inline .editorconfig]: add new line

File: src/App.tsx [hunk 2]
```diff
@@ -14,7 +15,7 @@ class App extends Component<IAppProps, IAppState> {
   public render() {
     return (
       <div className="App">
```
Comment (stanimirtt) [inline src/App.tsx]: Do we need that class?

File: src/components/Info/Avatar.scss [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+    border-radius: 50%;
+    max-width: 100%;
+    height: auto;
+    border: 10px solid #fff;
+    position: absolute;
+    top: 0;
+    left: 0;
+    margin: auto;
+    bottom: 0;
+
```
Comment (stanimirtt) [inline src/components/Info/Avatar.scss]: remove the empty line

**Review by dpyskunova — APPROVED** [review]

**Review by dpyskunova — APPROVED** [review]

**Review by stanimirtt — APPROVED** [review]

File: src/components/Info/Info.tsx [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+  <div className="info">
+    <div className="container">
+      <div className="info__grid">
+        <div className="info__description">
+          <p>Lorem ipsum dolor sit amet, consectetuer adipiscing elit.</p>
+        </div>
+        <Avatar source="https://api.adorable.io/avatars/285/abott@adorable.png" />
+
+        <div className="info__userName">
+          <h3 className="userName__name">John Smith</h3>
```
Comment (fstodulski) [inline src/components/Info/Info.tsx:17]: user-name__name instead of userName__name

**Review by fstodulski — APPROVED** [review]

**Pull request merged**